### DISCUSSION TITLE
Academic Reverification Required

### DISCUSSION THREAD

**Discussion by sinhaniik** [community/community#50316] [GitHub Education]
I am a member of __github education__ its been 3 days since I have been verified but now I am unable to get my dashboard of GitHub students where I used to have everything related to hackathons, GitHub partners etc 

I followed the step shown in the email i received gotten I submitted my college id card again but there is no response from GitHub it is accepted or rejected nothing 

any suggestions or help regarding this because I need to submit my octernship application soon and I am unable to do that

**Comment by samus-aran** [comment]
Hi @sinhaniik 

What errors are you having accessing Global Campus? Can you provide screenshots or error messages you are receiving.

**Comment by sinhaniik** [comment]
Hello @samus-aran it seems that this issue has been resolved yesterday, i have been reverified after all
